wagn/lib/wagn/generators/wagn/templates/config/application.rb

```diff
@@ -8,8 +8,9 @@ class Application < Wagn::Application
 
     # Wagn inherits Rails configuration options.  See http://guides.rubyonrails.org/configuring.html
 
-    config.recaptcha_public_key  = '6LdhRssSAAAAAFfLt1Wkw43hoaA8RTIgso9-tvtc'
-    config.recaptcha_private_key = '6LdhRssSAAAAAGwzl069pJQBdmzCZigm1nV-dmqK'
+    config.recaptcha_public_key  = '6LeoHfESAAAAAN1NdQeYHREq4jTSQhu1foEzv6KC'
+    config.recaptcha_private_key = '6LeoHfESAAAAAHLZpn7ijrO4_KGLEr2nGL4qjjis'
```

Comment:
If we're migrating these into cards anyway, shouldn't we just make these the default card content and get rid of the code configuration option?